lib/lookup_users.rb

```diff
@@ -2,6 +2,9 @@ module LookupUsers
 
   def self.by_email_or_username(email_or_username)
     email_or_username = email_or_username.try(:strip)
+
+    return [] if email_or_username.blank?
```

Comment:
This was the fix for the underlying problem causing us to errantly reach the `source = request.params["login"]["source"]` line below which gave a nil class error.